tests/tests_suite_test.go

```diff
@@ -0,0 +1,99 @@
+package tests
+
+import (
+	"fmt"
+	"context"
+	migapi "github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"path/filepath"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"testing"
+	// +kubebuilder:scaffold:imports
+)
+
+func TestMigmigration(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "E2E Suite")
+}
+
+var sourceClient *kubernetes.Clientset
+var k8sClient client.Client
+var cfg *rest.Config
+
+var _ = BeforeSuite(func() {
+	cfg, err := clientcmd.BuildConfigFromFlags("", filepath.Join("/home/jgabani/Downloads/jgabani0518/ocp4-workshop_jgabani0518_kubeconfig"))
```

Comment:
this should be env var?

Reply:
Yes! There is a lot of refactoring remaining.